evergreen/evergreen.yml

```diff
@@ -756,6 +757,26 @@ functions:
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
           $PYTHON -u kms_kmip_server.py
 
+  start-kms-mock-gcp-server:
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+          cd ${DRIVERS_TOOLS}/.evergreen/csfle
+          . ./activate_venv.sh
+    - command: shell.exec
+      params:
+        background: true
+        script: |
+          PYTHON=$(Venv="${DRIVERS_TOOLS}/.evergreen/csfle/kmstlsvenv" OS=${OS} ${PROJECT_DIRECTORY}/evergreen/get-python-path.sh);
+          cd ${DRIVERS_TOOLS}/.evergreen/csfle/gcpkms
+          $PYTHON -m pip install PyJWT
+          mkdir ${DRIVERS_TOOLS}/tmp
+          echo '${GOOGLE_APPLICATION_CREDENTIALS_CONTENT}' > ${DRIVERS_TOOLS}/tmp/testgcpkms_key_file.json
+          ls ${DRIVERS_TOOLS}/tmp/
```

Comment:
Do we still need to display the contents of `tmp` or was this only for debugging?

Reply:
true, removed